.pre-commit-config.yaml

```diff
@@ -0,0 +1,14 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+  - repo: https://github.com/norwoodj/helm-docs
+    rev: v1.13.1
+    hooks:
+      - id: helm-docs-container
```

Comment:
**issue:** I think we cannot use the container approach here, given the CI will not be able to run docker containers.

Reply:
I tried running the hook with both approaches (`helm-docs` and `container`). I removed my binaries and the hook failed, for both cases.

I opted for the container approach because I assumed one would more likely have docker installed than helm-docs. 

Why would the CI need to run the hook?

Reply:
> I removed my binaries and the hook failed, for both cases.
Yes, it's fine to expect the binary to be present on the machine (the dev machine or the CI).
It would be indeed nice to have the binary installed by pre-commit in a Python virtual env, but the binary is actually written in golang.

> Why would the CI need to run the hook?
The CI needs to check if the pre-commit hooks has run properly before comiting, and fail otherwise.

I think we can use the `helm-docs` hook which is safely checking the tool is install on the environment.
Also, we will need to add/install it to our Github action runner image.

Reply:
I also noticed that the repo provides a `helm-docs-built` here: https://github.com/norwoodj/helm-docs/blob/master/.pre-commit-hooks.yaml#L10.

This would build the tool from the source.
That being said, we need to make sure this has no much impact on the CI speed though.
The approach with the tool installed in the runner image might be preferable.